<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>1000</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>你画我猜</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.ico</normaloff>icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color:#FFFBFE;
}</string>
  </property>
  <widget class="QWidget" name="label">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="painter" name="myLabel" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>1000</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MiSans</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color:#FFFFFF;;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_requestPromote">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>10</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MiSans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:#6750A4;
border-style:outset;
border-width:3px;
border-radius:25px;
border-color:rgba(0,0,0,0);
color:#FFFFFF;
padding:12px;
}
QPushButton:hover{
background-color:#7057B3;
}
QPushButton:pressed{
background-color:#7D61C7;
}
</string>
    </property>
    <property name="text">
     <string>注册为Primary</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_requestProblem">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>110</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MiSans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:#6750A4;
border-style:outset;
border-width:3px;
border-radius:25px;
border-color:rgba(0,0,0,0);
color:#FFFFFF;
padding:12px;
}
QPushButton:hover{
background-color:#7057B3;
}
QPushButton:pressed{
background-color:#7D61C7;
}
</string>
    </property>
    <property name="text">
     <string>随机抽题</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>900</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { color : #6750A4; }</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_connection">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>870</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { color : #6750A4; }</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_promotion">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>870</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { color : #6750A4; }</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clearPaint">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>160</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MiSans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:#6750A4;
border-style:outset;
border-width:3px;
border-radius:25px;
border-color:rgba(0,0,0,0);
color:#FFFFFF;
padding:12px;
}
QPushButton:hover{
background-color:#7057B3;
}
QPushButton:pressed{
background-color:#7D61C7;
}
</string>
    </property>
    <property name="text">
     <string>清空画布</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_problem">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>230</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MiSans</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { color : #6750A4; }</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_countdown">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>290</y>
      <width>141</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MiSans</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { color : #6750A4; }</string>
    </property>
    <property name="text">
     <string>45</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_export">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>60</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MiSans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:#6750A4;
border-style:outset;
border-width:3px;
border-radius:25px;
border-color:rgba(0,0,0,0);
color:#FFFFFF;
padding:12px;
}
QPushButton:hover{
background-color:#7057B3;
}
QPushButton:pressed{
background-color:#7D61C7;
}
</string>
    </property>
    <property name="text">
     <string>导出图像</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="answer">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>610</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MiSans</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="lineWrapMode">
     <enum>QPlainTextEdit::NoWrap</enum>
    </property>
    <property name="maximumBlockCount">
     <number>4</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_submit">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>650</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MiSans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:#EADDFF;
border-style:outset;
border-width:3px;
border-radius:25px;
border-color:rgba(0,0,0,0);
color:#21005D;
padding:12px;
}
QPushButton:hover{
background-color:#DFD3F2;
}
QPushButton:pressed{
background-color:#D3C8E6;
}
</string>
    </property>
    <property name="text">
     <string>提交答案！</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_color">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>400</y>
      <width>111</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>590</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>painter</class>
   <extends>QWidget</extends>
   <header location="global">painter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
